pkg/client/cli/license.go

```diff
@@ -0,0 +1,40 @@
+package cli
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/telepresenceio/telepresence/v2/pkg/client/cli/cliutil"
+)
+
+type licenseInfo struct {
+	outputFile string
+}
+
+func LicenseCommand() *cobra.Command {
+	li := &licenseInfo{}
+	cmd := &cobra.Command{
+		Use:  "license",
+		Args: cobra.MinimumNArgs(1),
```

Comment:
If you're going to accept positional arguments, the `Use` string should tell the user what they are.

For example, `intercept_state.go`'s
```go
Use:  "intercept [flags] <intercept_base_name> [-- <command with arguments...>]",
Args: cobra.MinimumNArgs(1),
```

Reply:
And it looks like it expects exactly 1 arg, not ≥1 args, so this should be `cobra.ExactArgs(1)`.